gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/saml/SamlSsoConfig.groovy

```diff
@@ -155,6 +156,12 @@ class SamlSsoConfig extends WebSecurityConfigurerAdapter {
         def email = assertion.getSubject().nameID.value
         def roles = extractRoles(email, attributes, userAttributeMapping)
 
+        if (samlSecurityConfigProperties.requiredRoles) {
+          if (!roles.containsAll(samlSecurityConfigProperties.requiredRoles)) {
```

Comment:
Correct, I checked back with the old code & had mis-remembered. I updated the PR.